Sorts/test/MergeSort.test.ts

```diff
@@ -1,20 +1,12 @@
 import { MergeSort } from "../MergeSort";
 
-/**
- * Space complexity - O(n)
- * Time complexity 
- *      Best case   -   O(nlogn)
- *      Worst case  -   O(nlogn)
- *      Average case -  O(nlogn)
- * 
- * Merge Sort is a recursive algorithm and time complexity can be expressed as following recurrence relation. 
- * T(n) = 2T(n/2) + O(n) 
- * The solution of the above recurrence is O(nLogn).
- */
-
 describe("Merge Sort", () => {
-    it("should return the correct value", () => {
-        expect(MergeSort([1, 4, 2, 5, 9, 6, 3, 8, 10, 7])).toStrictEqual([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]);
-    });
+    it("generating array and comparing with sorted array", () => {
+        let arrLen = 10;
+        let inBuiltSortArr = Array.from(Array(arrLen)).map(x=>Math.random());
```

Comment:
Please stick this in a for loop (and also test longer arrays)

Reply:
tested and updated